src/cross_section/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 The Manifold Authors.
```

Comment:
Let's update these dates to 2023.

Reply:
There's a few more of them...

Reply:
Oh right... I'll do a replace. For some reason I interpreted as just on this new one. :sweat_smile: